cloudant-client/src/test/java/com/cloudant/tests/extensions/CloudantClientMockServerExtension.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2017 IBM Corp. All rights reserved.
```

Comment:
As you said in the TODOs description for this PR there are a bunch of copyright updates to do.